### PR TITLE
coreos/azure: Updates for Weave 1.1

### PR DESCRIPTION
**Pull request by errordeveloper**
- Bump CLI tools version
- Bump CoreOS image IDs

### CODE REVIEW

**Comment by errordeveloper** [comment]
Most significant change here is the switch to Weave IPAM and Docker API proxy.

**Comment by brendandburns** [comment]
This looks reasonable.  I re-ran shippable which looks like a flake.

**Comment by errordeveloper** [comment]
Rebased.

File: docs/getting-started-guides/coreos/azure/cloud_config_templates/kubernetes-cluster-main-nodes-template.yml [hunk 1]
```diff
@@ -191,7 +184,7 @@ coreos:
         Documentation=http://kubernetes.io/
         Requires=network-online.target
         [Service]
-        Environment=KUBE_RELEASE_TARBALL=https://github.com/GoogleCloudPlatform/kubernetes/releases/download/v1.0.1/kubernetes.tar.gz
+        Environment=KUBE_RELEASE_TARBALL=https://github.com/GoogleCloudPlatform/kubernetes/releases/download/v1.0.6/kubernetes.tar.gz
```
Comment (errordeveloper) [inline docs/getting-started-guides/coreos/azure/cloud_config_templates/kubernetes-cluster-main-nodes-template.yml]: I have tested `v1.1.0-alpha.1` also, and it works okay too.

**Pull request merged**